public/app/features/explore/Explore.tsx

```diff
@@ -52,6 +53,16 @@ import { ErrorContainer } from './ErrorContainer';
 import { scanStopAction } from './state/actionTypes';
 import { ExploreGraphPanel } from './ExploreGraphPanel';
 
+const getStyles = memoizeOne(() => {
```

Comment:
Out of curiosity why do we use `memoizeOne` for getStyles?

Reply:
There was a discussion in some PR where to put styles creation. I did not like having it in render as they usually did not change between rerenders and are a bit distracting there imho. @torkelo on the other hand did not like that styles get created in module scope, creating stuff implicitly during import. So having a memoized function seems like a good solution for that as it is created only once and is out of render. On the other hand it is more of personal preference I think.

Reply:
Thanks, I guess we should talk about this at FrontEnd Bi-Weekly, seems like we should agree on a common approach :)

Reply:
Yes good point, I see you already added it to agenda 👍